crud/common/utils.lua

```diff
@@ -375,6 +384,74 @@ function utils.get_bucket_id_fieldno(space, shard_index_name)
     return bucket_id_index.parts[1].fieldno
 end
 
+function utils.get_index_fieldno_map(index_obj)
+    dev_checks('table')
+
+    local fieldno_map = {}
+    for _, part in ipairs(index_obj.parts) do
+        local fieldno = part.fieldno
+        fieldno_map[fieldno] = true
+    end
+
+    return fieldno_map
+end
+
+function utils.get_format_fieldno_map(space_format)
+    dev_checks('table')
+
+    local fieldno_map = {}
+    for fieldno, field_format in ipairs(space_format) do
+        fieldno_map[field_format.name] = fieldno
+    end
+
+    return fieldno_map
+end
+
+--- Get a map with fieldno of passed field's names.
+--
+-- @function get_keys_fieldno_map
+--
+-- @param table space_format
+--  A space format
+--
+-- @param keys
+--  A table with field names.
+--
+-- @return[1] table
+-- @return[2] nil
+--
+function utils.get_keys_fieldno_map(space_format, field_names)
+    dev_checks('table', 'table')
+
+    local t = {}
+    local fieldno_map = utils.get_format_fieldno_map(space_format)
+    for _, field_name in ipairs(field_names) do
+        local fieldno = fieldno_map[field_name]
+        if fieldno == nil then
+            return nil
+        end
+        t[fieldno] = true
+    end
+
+    return t
+end
+
+function utils.get_keys_fieldnos(space_format, field_names)
```

Comment:
keys fieldnos could be easily exctacted from the index. Because index.parts has the same order as key.

The single difference - key length is less or equal than index parts count.

Reply:
What for do we need such changes? I don't understand idea behind your suggestion.

Reply:
`get_keys_fieldnos()` has been removed